doc-content/enterprise-only/optimizer/vaccination-scheduler-download-proc.adoc

```diff
@@ -10,8 +10,8 @@ Download the OptaPlanner vaccination appointment scheduler quick start archive,
 
 ** Product: {PRODUCT_SHORT}
 ** Version: {PRODUCT_VERSION}
-. Download *{PRODUCT} {PRODUCT_VERSION_LONG} Kogito and OptaPlanner 8 Decision Services Quickstarts* (`{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-decision-services-quickstarts.zip`).
-. Extract the `{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-decision-services-quickstarts.zip` file.
+. Download *{PRODUCT} {PRODUCT_VERSION_LONG} Kogito and OptaPlanner 8 Decision Services Quickstarts* (`{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-kogito-and-optaplanner-quickstarts.zip`).
+. Extract the `{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-kogito-and-optaplanner-quickstarts.zip` file.
 . Navigate to the `optaplanner-quickstarts-{OPTAPLANNER_BOM_VERSION}` directory.
 . Navigate to the `optaplanner-quickstarts-{OPTAPLANNER_BOM_VERSION}/quarkus-vaccination-scheduling` directory.
```

Comment:
Directory should be 
optaplanner-quickstarts-8.11.1.Final-redhat-00003/use-cases/vaccination-scheduling